android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt

```diff
@@ -96,6 +103,11 @@ constructor(
       .searchCompositionByIdentifier(appId)
       .also { if (it == null) configsLoadedCallback(false) }
       ?.section
+      ?.filter {
+        it.focus.reference.split("/")[0].let { resourceType ->
+          resourceType == ResourceType.Parameters.name || resourceType == ResourceType.Binary.name
+        }
+      }
       ?.forEach {
         // each section in composition represents workflow
         // { "title": "register configuration",
```

Comment:
@roywanyaga Remove dead code

Reply:
This has been updated. The comments have been provided as javadoc